src/main/kotlin/controller/RacingCarGame.kt

```diff
@@ -0,0 +1,27 @@
+package controller
+
+import model.RacingCar
+import view.InputView
+import view.ResultView
+
+fun main() {
+    RacingCarGame(InputView(), ResultView()).run()
+}
+
+class RacingCarGame(private val inputView: InputView, private val resultView: ResultView) {
+    fun run() {
+        val cars = inputView.enterAmountOfCars()
+        val rounds = inputView.enterAmountOfRounds()
+
+        val racingCars: List<RacingCar> = List(cars) { RacingCar() }
+        resultView.printRaceResultMessage()
+        repeat(rounds) {
+            resultView.printRacingCarGameState(racingCars)
+            moveRacingCars(racingCars)
```

Comment:
How about showing the final result only once, after the main logic is done?
It can help make the return values cleaner and support a better understanding of the MVC pattern.